app/src/androidTest/java/ca/marshallasch/veil/MainUITest.java

```diff
@@ -0,0 +1,267 @@
+package ca.marshallasch.veil;
```

Comment:
Can you please add comments to this class because it is pretty large and not that easy to follow,